### ISSUE TITLE
Trying to use the Wireless Crafting Grid from RS crashes the game

### ISSUE THREAD

**Issue by SamayaNL** [Iskallia/Vault-public#816]
**Describe the bug**
Trying to use the Wireless Crafting Grid from RS crashes the game

**To Reproduce**
1. Craft a Wireless Crafting Grid
2. Charge it
3. Link it to network by shift right clicking it to the controller
4. Right click the WCG while in hotbar

**Expected behavior**
The crafting grid opens

**Screenshots or Crash Report**
[crash-2022-02-17_01.40.36-server.txt](https://github.com/Iskallia/Vault-public-S1/files/8084630/crash-2022-02-17_01.40.36-server.txt)

**Info (please complete the following information):**
- World Type: Single Player
- The_Vault Version 1.12.4

**Comment by DarkPhoenix1986** [comment]
This just happened to me after updating from VH 1.12.1 to 1.12.4. Is there a fix for this? I also tried re-linking the wireless crafting Grid to the system after logging back in and it crashed again.
[crash-2022-05-04_19.25.32-server.txt](https://github.com/Iskallia/Vault-public-S1/files/8628460/crash-2022-05-04_19.25.32-server.txt)

**Comment by DarkPhoenix1986** [comment]
As a followup I tried to give myself a different Wireless Crafting Grid and when trying to link it to the network it crashed. 

The game crashed whilst exception in server tick loop
Error: java.lang.AbstractMethodError: Method com/refinedmods/refinedstorageaddons/item/WirelessCraftingGridItem.provide(Lcom/refinedmods/refinedstorage/api/network/item/INetworkItemManager;Lnet/minecraft/entity/player/PlayerEntity;Lnet/minecraft/item/ItemStack;Lcom/refinedmods/refinedstorage/inventory/player/PlayerSlot;)Lcom/refinedmods/refinedstorage/api/network/item/INetworkItem; is abstract
Exit Code: -1
[crash-2022-05-06_16.02.20-server.txt](https://github.com/Iskallia/Vault-public-S1/files/8643641/crash-2022-05-06_16.02.20-server.txt)

**Comment by SamayaNL** [comment]
> As a followup I tried to give myself a different Wireless Crafting Grid and when trying to link it to the network it crashed.
> 
> The game crashed whilst exception in server tick loop Error: java.lang.AbstractMethodError: Method com/refinedmods/refinedstorageaddons/item/WirelessCraftingGridItem.provide(Lcom/refinedmods/refinedstorage/api/network/item/INetworkItemManager;Lnet/minecraft/entity/player/PlayerEntity;Lnet/minecraft/item/ItemStack;Lcom/refinedmods/refinedstorage/inventory/player/PlayerSlot;)Lcom/refinedmods/refinedstorage/api/network/item/INetworkItem; is abstract Exit Code: -1 [crash-2022-05-06_16.02.20-server.txt](https://github.com/Iskallia/Vault-public-S1/files/8643641/crash-2022-05-06_16.02.20-server.txt)

What solved the issue for me was manually updating the RS mod, and the RS add-ons mod (not sure what it's called). I don't know which one fixed the issue, but it did.

**Comment by DarkPhoenix1986** [comment]
> > As a followup I tried to give myself a different Wireless Crafting Grid and when trying to link it to the network it crashed.
> > The game crashed whilst exception in server tick loop Error: java.lang.AbstractMethodError: Method com/refinedmods/refinedstorageaddons/item/WirelessCraftingGridItem.provide(Lcom/refinedmods/refinedstorage/api/network/item/INetworkItemManager;Lnet/minecraft/entity/player/PlayerEntity;Lnet/minecraft/item/ItemStack;Lcom/refinedmods/refinedstorage/inventory/player/PlayerSlot;)Lcom/refinedmods/refinedstorage/api/network/item/INetworkItem; is abstract Exit Code: -1 [crash-2022-05-06_16.02.20-server.txt](https://github.com/Iskallia/Vault-public-S1/files/8643641/crash-2022-05-06_16.02.20-server.txt)
> 
> What solved the issue for me was manually updating the RS mod, and the RS add-ons mod (not sure what it's called). I don't know which one fixed the issue, but it did.

Did you just download the updated RS mods and replace in the folder?

**Comment by SamayaNL** [comment]
> > > As a followup I tried to give myself a different Wireless Crafting Grid and when trying to link it to the network it crashed.
> > > The game crashed whilst exception in server tick loop Error: java.lang.AbstractMethodError: Method com/refinedmods/refinedstorageaddons/item/WirelessCraftingGridItem.provide(Lcom/refinedmods/refinedstorage/api/network/item/INetworkItemManager;Lnet/minecraft/entity/player/PlayerEntity;Lnet/minecraft/item/ItemStack;Lcom/refinedmods/refinedstorage/inventory/player/PlayerSlot;)Lcom/refinedmods/refinedstorage/api/network/item/INetworkItem; is abstract Exit Code: -1 [crash-2022-05-06_16.02.20-server.txt](https://github.com/Iskallia/Vault-public-S1/files/8643641/crash-2022-05-06_16.02.20-server.txt)
> > 
> > 
> > What solved the issue for me was manually updating the RS mod, and the RS add-ons mod (not sure what it's called). I don't know which one fixed the issue, but it did.
> 
> Did you just download the updated RS mods and replace in the folder?

Into the mods folder yup

**Comment by DarkPhoenix1986** [comment]
So for anyone looking for the fix, I updated Refined Storage mod to v1.9.17, which still didn't fix the problem. I updated Refined Storage Addon mod (which I now believe is the culprit, see error code posted above where it mentions "refinedstorageaddons".) I updated it to v0.7.4 and it now works. 

TLDR: Try updating Refined Storage Addons mod to v0.7.4 (or other appropriate versions).
